server/util/orphaned-records-utils.ts

```diff
@@ -71,9 +74,8 @@ export function buildVisibleOrphanedRecordResultsQuery(userEdipi: string, orgId:
     .addSelect('orphan.composite_id', 'id')
     .addSelect('roster.unit_id', 'unitId')
     .addSelect('roster.id', 'rosterHistoryId')
-    .orderBy('orphan.count', 'DESC')
-    .addOrderBy('orphan.unit', 'ASC')
-    .addOrderBy('orphan.edipi', 'ASC')
+    .orderBy('orphan.unit', 'ASC')
+    .addOrderBy('MAX(orphan.timestamp)', 'DESC')
```

Comment:
This groups by unit first since their workflow is reportedly by unit. And the `MAX()` is there to get the latest timestamp for the current grouped aggregate (the value that shows up as the "Last Reported On" column in the orphaned record UI.).  This really isn't part of the current PR but it was done today with Austin and Bingles based on their feedback from the Lt.